src/components/SignIn.jsx

```diff
@@ -48,29 +54,51 @@ class SignIn extends React.Component {
 
   render() {
     const { email, password } = this.state;
+    const { classes } = this.props;
     return (
-      <form>
-        <DialogInput
-          value={email}
-          name="email"
-          label="Email"
-          onChange={this.handleChange} />
-        <DialogInput
-          type="password"
-          value={password}
-          name="password"
-          label="Password"
-          onChange={this.handleChange} />
-        <br />
-        <Button buttonType="button" onClick={this.handleSubmit} type="default">
-          Submit
-        </Button>
+      <form className={classes.loginSectionStyle}>
+        <div style={{textAlign: "center", paddingTop: "50px"}}>
+          <h4 style={{fontFamily: "Fjalla One", color: "white", fontSize: "25px"}}>
+            OSSDF DANCEFEST
+          </h4>
+        </div>
+        <div style={{marginTop: "100px", marginLeft: "139px"}}>
+          <DialogInput
+            className={classes.loginDialogInput}
+            value={email}
+            name="email"
+            InputProps={{
+              classes: {
+                input: classes.multilineColor,
+              }
+            }}
+            label={inputLabel("Email")}
+            onChange={this.handleChange} />
+          <br />
+          <DialogInput
+            className={classes.loginDialogInput}
+            type="password"
+            value={password}
+            name="password"
+            InputProps={{
```

Comment:
extract this into a  external class as well, we can reuse these objects between this one and the one on line 70.